client/landing/gutenboarding/onboarding-block/create-site-error/index.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent, useEffect } from 'react';
+import { useI18n } from '@automattic/react-i18n';
+import { ExternalLink } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import Link from '../../components/link';
+import './style.scss';
+import { createInterpolateElement } from '@wordpress/element';
```

Comment:
Tiniest of nits: these `@wordpress` imports should be in the `External dependencies` section above.